### PR TITLE
Fix display of search results in IE 11. Fixes issue #99.

### PR DESCRIPTION
**Pull request by craiga**
### Fix display of search results in IE 11

Fixes https://github.com/MHRA/products/issues/99

giphy-9

### Acceptance Criteria

- [ ] Search results are laid out in IE 11 similarly to how they're laid out in other modern browsers.

### Testing information

<insert notes for testers that might be helpful>

### Pre-flight Checklist

- [ ] Tests
- [ ] DUXD review approval
- [ ] Accessibility Reviewed
- [ ] Product owner demo

### Pre-merge Checklist

- [ ] Branch test approved

### CODE REVIEW

File: medicines/web/components/search-results/index.tsx [hunk 1]
```diff
@@ -221,8 +221,8 @@ const SearchResults = (props: {
           )}
           <p className="ema-message">
             If the product information you are seeking does not appear below, it
-            is possible that the product holds a European licence and
-            its information may be available at the {emaWebsiteLink()} website.
```
Comment (craiga) [inline medicines/web/components/search-results/index.tsx]: `tslint` made me move "its".

Comment (callerc1) [reply medicines/web/components/search-results/index.tsx]: it's ?

Comment (craiga) [reply medicines/web/components/search-results/index.tsx]: @callerc1 I'm pretty sure "its" is correct.

"it's" is a contraction of "it is", which would mean the sentence is saying "…the product holds a European licence and _it is_ information may be available…"

Comment (callerc1) [reply medicines/web/components/search-results/index.tsx]: cool 👍

**Pull request merged**